testdata/metric/client_request_messages.yaml.tmpl

```diff
@@ -0,0 +1,52 @@
+name: istio_request_messages
+type: COUNTER
+metric:
+- counter:
+    value: {{ .Vars.RequestMessages }}
+  label:
+  - name: reporter
+    value: source
+  - name: source_workload
+    value: productpage-v1
+  - name: source_canonical_service
+    value: productpage-v1
+  - name: source_canonical_revision
+    value: version-1
+  - name: source_workload_namespace
+    value: default
+  - name: source_principal
+    value: unknown
+  - name: source_app
+    value: productpage
+  - name: source_version
+    value: v1
+  - name: destination_workload
+    value: ratings-v1
+  - name: destination_workload_namespace
+    value: default
+  - name: destination_principal
+    value: unknown
+  - name: destination_app
+    value: ratings
+  - name: destination_version
+    value: v1
+  - name: destination_service
+    value: server.default.svc.cluster.local
+  - name: destination_canonical_service
+    value: ratings
+  - name: destination_canonical_revision
+    value: version-1
+  - name: destination_service_name
+    value: server
+  - name: destination_service_namespace
+    value: default
+  - name: request_protocol
```

Comment:
i wonder, is this useful?  will these metrics already imply grpc?

Reply:
Yes. The same problem is with TCP (it's also tcp there). It's not useful.

Reply:
Removed.